### PR TITLE
[DC-1224] Integration Test for Gender/Sex Mismatch CR

### PR DESCRIPTION
**Pull request by lledezma**

### CODE REVIEW

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generalize_sex_gender_concepts_test.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+GENERALIZED_CONCEPT_ID_TEST_QUERY_TEMPLATE = JINJA_ENV.from_string("""
+INSERT INTO `{{project_id}}.{{dataset_id}}.observation`
+
+(observation_id, person_id, observation_concept_id, observation_date, observation_datetime, 
+observation_type_concept_id, value_as_number, value_as_string, value_as_concept_id, qualifier_concept_id, 
+unit_concept_id, provider_id, visit_occurrence_id, visit_detail_id, observation_source_value,
+observation_source_concept_id,unit_source_value,qualifier_source_value,value_source_concept_id,value_source_value,questionnaire_response_id)
+
+VALUES
+    (100, 1, 1585838, '2009-04-29', TIMESTAMP('2009-04-29'),
```
Comment (jp3477) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generalize_sex_gender_concepts_test.py:29]: Why are some of these `observation_id`(s) duplicated in the test records? (100 & 300)

**Review by jp3477 — CHANGES_REQUESTED** [review]
Hi Luis, this looks good for the most part. You'll just need to add the rule to the clean_cdr under the registered tier deid, I believe. Also, I left a comment about duplicated observation_ids in your tests.

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generalize_sex_gender_concepts_test.py (hunk 1 above)
Comment (lledezma) [reply tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generalize_sex_gender_concepts_test.py:30]: I forgot to update the test records.

**Review by jp3477 — APPROVED** [review]
Approved!

**Pull request merged**